src/posix/net.cpp

```diff
@@ -306,7 +306,10 @@ CurlHttpIO::CurlHttpIO()
     options.tries = 2;
     ares_init_options(&ares, &options, ARES_OPT_TRIES);
     arestimeout = -1;
+    
+#ifndef TARGET_OS_IPHONE
```

Comment:
If we don't want to filter DNS servers on iOS (which I think it's useless too), maybe it's cleaner to change the method, rather than the calls (ie. add a return inside `filterDNSservers()` in case of iOS).

Reply:
Done!